### DISCUSSION TITLE
Welcome to cosmichub Discussions!

### DISCUSSION THREAD

**Discussion by GodOfCode00** [GodOfCode00/cosmichub#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GodOfCode00** [comment]
Shall I include the frontend on this repo or in a separate repo ?

**Reply by biswasray** [reply]
We should include the frontend in this repo

**Comment by biswasray** [comment]
We should make its structure standardised
